src/dmtcpplugin.cpp

```diff
@@ -199,7 +199,10 @@ EXTERNC int dmtcp_set_ckpt_dir(const char* dir)
 EXTERNC const char* dmtcp_get_coord_ckpt_dir(void)
```

Comment:
Could you please document that the API returns a zero-length string, "", when a checkpoint is in progress?